full/src/test/java/apoc/metrics/MetricsTest.java

```diff
@@ -62,6 +68,32 @@ public void shouldGetMetrics() {
                 });
     }
 
+    @Test
+    public void shouldRetrieveStorageMetrics() {
+
+        final Set<String> expectedSet = Stream.of("setting", "freeSpaceBytes", "totalSpaceBytes", "usableSpaceBytes", "percentFree")
+                .collect(Collectors.toSet());
+
+        NEO4J_DIRECTORY_CONFIGURATION_SETTING_NAMES.forEach(setting ->
+                testCall(session, "CALL apoc.metrics.storage($setting)", map("setting", setting), (res) -> {
+                    assertEquals(expectedSet, res.keySet());
+                    assertEquals(setting, res.get("setting"));
+                })
+        );
+
+        // all metrics with null
+        testResult(session, "CALL apoc.metrics.storage(null)", (res) -> {
```

Comment:
out of curiosity, why `CALL apoc.metrics.storage(null)` and not `CALL apoc.metrics.storage()`?